src/Views/Games/Sudoku/Sudoku.js

```diff
@@ -128,7 +41,16 @@ function Sudoku() {
     };
 
     useEffect(() => {
-        setBoard(getBoard());
+        fetch("http://127.0.0.1:5000/sudoku?difficulty=medium", {
```

Comment:
this needs to be the correct URL and difficulty should be a variable.

Reply:
How do I get the correct URL?

Reply:
api.speakeasy.services/sudoku?difficulty=medium